tests/test_pipeline.py

```diff
@@ -216,3 +218,14 @@ def test_pipeline_changing_workflow():
     expected = np.array([[1., 0.30384404],
                          [0.30384404, 1.]])
     assert np.allclose(all_scores["ModifiedCosine_score"][3:, 3:], expected)
+
+
+def test_save_spectra_spectrum_processor():
```

Comment:
You could use the tmp_path fixture from pytest to simplify the setup. See https://docs.pytest.org/en/7.4.x/how-to/tmp_path.html

Reply:
Indeed. We still have several of those older "with tempfile... " constructions in our tests. Probably can simply be replaced by `tmp_dir` (tryout #566).

Reply:
Thanks. Not sure why I did not go for tmp_path this time...